features/bootstrap/JsonContext.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Mink\Exception\ExpectationException;
+use Behatch\Context\JsonContext as BehatchJsonContext;
+use Coduo\PHPMatcher\PHPMatcher;
+
+class JsonContext extends BehatchJsonContext
+{
+    public function theJsonShouldBeEqualTo(PyStringNode $content)
+    {
+        $actual = $this->getJson();
```

Comment:
This declaration variable is not really needed

Reply:
agree thank you. I'll fix it in WebHook Pr.